exporter/logtospanexporter/config.go

```diff
@@ -0,0 +1,118 @@
+package logtospanexporter
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/consumer/consumererror"
+	otlp "go.opentelemetry.io/collector/exporter/otlpexporter"
+)
+
+// Config defines configuration for the Log to Span exporter.
+type Config struct {
+	// Config represents the OTLP Exporter configuration.
+	otlp.Config `mapstructure:",squash"`
+
+	// FieldMap represents the fields that are mapped from the log to the outbound span.
+	FieldMap FieldMap `mapstructure:"field_map"`
+}
+
+// FieldMap defines the log fields that map to span fields (case sensitive).
+type FieldMap struct {
+	// TraceType related configuration
+	// Type of span that the log represents. (default w3c)
+	//   * w3c - Logs contain W3C Trace Context 'traceparent' and 'tracestate' attributes
+	TraceType TraceType `mapstructure:"trace_type"`
+
+	// W3CTraceContextFields defines the log fields used for the W3C Trace Context
+	W3CTraceContextFields W3CTraceContextFields `mapstructure:"w3c"`
+
+	// SpanName defines the name of the span.
+	SpanName string `mapstructure:"span_name"`
```

Comment:
Would it make sense to treat SpanName like the others and derive it from a specified attribute (maybe with an optional default)?

Reply:
That's exactly what it is; however, the comment is incorrect. 

I also thought about having a default, but I'm not sure a default would ever be used...

Reply:
haha sorry that was obvious from context I just didn't catch it. Agreed probably no need for default.

Reply:
No need to apologize. It helped catch the terrible comment :)